crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -86,50 +89,39 @@ pub fn cli() -> clap::Command {
                 .value_parser(clap::value_parser!(Language))
                 .help("The language to generate"),
         )
-        .arg(
-            Arg::new("skip_clippy")
-                .long("skip_clippy")
-                .short('s')
-                .short('S')
-                .action(SetTrue)
-                .env("SPACETIME_SKIP_CLIPPY")
-                .value_parser(clap::builder::FalseyValueParser::new())
-                .help(
-                    "Skips running clippy on the module before generating \
-                     (intended to speed up local iteration, not recommended \
-                     for CI)",
-                ),
-        )
```

Comment:
It's a little unfortunate if we remove this, this option specifically saves me quite a lot of time as a user of SpacetimeDB. Which like it's probably worth it to have a conversation about clippy. The times I've shot myself in the foot from misusing `println!` instead of `log:info!` or whatever is IMO relatively small compared to the amount of time I would have spent waiting for clippy to run. In ephemeral projects where I don't have a full "build/publish" script put together I commonly use this option to speed up the process.

No action is needed here for now - we can remove it from now and see how bad it is or how many complaints we get and if we get to that point maybe we consider adding this back in.

Reply:
Just to be clear, we still have this flag - it just needs to be passed as a build option instead of directly.

or are you talking about the env var?